.github/workflows/test.yml

```diff
@@ -370,7 +393,8 @@ jobs:
 
       - name: Downcase network name for disks
         run: |
-          echo LOWER_NET_NAME="${{ github.event.inputs.network || env.NETWORK }}" | awk '{print tolower($0)}' >> $GITHUB_ENV
+          echo NETWORK_CAPS="${{ github.event.inputs.network || env.NETWORK }}" | awk '{print tolower($0)}'
```

Comment:
Opposite of CAPS, it's lowercase

Reply:
Fixed by: [fix(ci): use a better name for network string conversion](https://github.com/ZcashFoundation/zebra/pull/4073/commits/2d026be71d2a472252464353473a100504de5f27)